### PR TITLE
:stethoscope: Add health check endpoint

### PR DESCRIPTION
**Pull request by PaulMathon**

### CODE REVIEW

**Review by tibonihoo — APPROVED** [review]
Good for me to get a stub health check end point.

It seems the health check is not checking much so we should start (for another review) thinking about more meaningful checks. When looking at how the app is implemented, we could for instance benefit from trying to see if it can create a report folder (or something inside it) as a cheap test for access rights and maybe space and also trying to call newman itself (maybe by asking the version if that's possible) ?

File: app.js [hunk 1]
```diff
@@ -21,6 +22,7 @@ class Application{
         url: '/openapi.yaml'
       }
     }
+    expressApp.use('/api/health', healthCheck());
     expressApp.use('/api-docs', swaggerUi.serve, swaggerUi.setup(null, options));
```
Comment (ouvreboite) [inline app.js:26]: Can we align the endpoints here? /api/health & /api/docs?

**Review by ouvreboite — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (PaulMathon) [reply app.js:26]: of course!

**Review by tibonihoo — APPROVED** [review]
Good for me.

**Review by tibonihoo — APPROVED** [review]

**Review by ouvreboite — APPROVED** [review]

**Pull request merged**